### DISCUSSION TITLE
cartella conf

### DISCUSSION THREAD

**Discussion by fireport** [Home-Assistant-Pro-Team/Gestione-Allarme-Home-Assistant-Alarm#3] [Q&A]
Buongiorno a tutti.
La documentazione dice che bisogna copiatre i template jinja nella carella conf. Sapete dirmi dove si trova ?
Grazie

**Comment by marco-hacs** [comment]
era un errore di battitura... la cartella è "config". la stessa dove risiedono i file di configurazione

**Comment by marco-hacs** [comment]
.
